tests/behat/features/gridfield-navigation.feature

```diff
@@ -0,0 +1,160 @@
+Feature: Search in GridField
+  As an author
+  I want to navigate through the filtered and sorted GridField list
+  So that I see the filtered and sorted GridField when I navigate to an item and back to the GridFIeld list  
+
+  Background:
+    Given the "Company" "Walmart" with "Category"="Retail"
+      And the "Employee" "Alen" with "Company"="1"
+      And the "Employee" "Bill" with "Company"="1"
+      And the "Employee" "Ford" with "Company"="1"
+      And the "Company" "ExxonMobil" with "Category"="Oil"
+      And the "Company" "Vitol" with "Category"="Other"
+      And the "Company" "Safeway" with "Category"="Other" 
+      And the "Company" "SuperValue" with "Category"="Other" 
+      And the "Company" "UBS" with "Category"="Other"
+      And the "Company" "Kraft Foods" with "Category"="Other" 
+      And the "Company" "Ahold" with "Category"="Other"
+      And the "Company" "Cisco" with "Category"="Other"
+      And the "Company" "Bouygues" with "Category"="Other" 
+      And the "Company" "Insurance" with "Category"="Other"
+      And I am logged in with "ADMIN" permissions
+
+  Scenario: I can navigate back through the GridField items by using better button
```

Comment:
```suggestion
  Scenario: I can navigate back through the GridField items by using the "previous record" button
```
The term "better buttons" refers to an old module which use to be used instead of the core functionality. We should avoid using that terminology in the actual code and tests.

Reply:
DONE